packages/fuel-gauge/src/predicate/predicate-configurables.test.ts

```diff
@@ -187,52 +196,60 @@ describe('Predicate', () => {
       ).rejects.toThrow(/PredicateVerificationFailed/);
     });
 
-    it('throws when setting configurable but predicate has none', () => {
-      expect(() => {
-        // eslint-disable-next-line @typescript-eslint/no-unused-vars
-        const predicate = new Predicate({
-          bytecode: predicateBytesTrue,
-          abi: predicateAbiTrue,
-          provider: wallet.provider,
-          inputData: ['NADA'],
-          configurableConstants: {
-            constant: 'NADA',
-          },
-        });
-      }).toThrow('Predicate has no configurable constants to be set');
+    it('throws when setting configurable but predicate has none', async () => {
+      await expectToThrowFuelError(
+        () =>
+          new Predicate({
+            bytecode: predicateBytesTrue,
+            abi: predicateAbiTrue,
+            provider: wallet.provider,
+            inputData: ['NADA'],
+            configurableConstants: {
+              constant: 'NADA',
+            },
+          }),
+        {
+          code: ErrorCode.INVALID_CONFIGURABLE_CONSTANTS,
+          message: expect.stringContaining('Predicate has no configurable constants to be set'),
+        }
```

Comment:
Do not pass this object but rather 
```ts
new FuelError(ErrorCode.INVALID_CONFIGURABLE_CONSTANTS, "Predicate has no configurable constants to be set")
```

Reply:
I don't know why, but the test isn't passing

Reply:
@nedsalk look at here:
```ts
- [FuelError: Predicate has no configurable constants to be set]
+ Object {
+   "VERSIONS": Object {
+     "FORC": "0.60.0",
+     "FUELS": "0.90.0",
+     "FUEL_CORE": "0.30.0",
+   },
+   "code": "invalid-configurable-constants",
+   "metadata": Object {},
+   "name": "FuelError",
+ }

Reply:
I don't see any failing tests; CI is green for all workflows.

But as @nedsalk said, you should not assert the error with an Object:
https://github.com/FuelLabs/fuels-ts/blob/06aadf6e1efa1bd8701c0e55e18f99d81aee9263/packages/fuel-gauge/src/predicate/predicate-configurables.test.ts#L211-L214

Instead, you should use `FuelError`, as you did on the other occurrences:

https://github.com/FuelLabs/fuels-ts/blob/06aadf6e1efa1bd8701c0e55e18f99d81aee9263/packages/fuel-gauge/src/predicate/predicate-configurables.test.ts#L230-L233

Reply:
> I don't see any failing tests; CI is green for all workflows.
> 
> But as @nedsalk said, you should not assert the error with an Object:
> 
> https://github.com/FuelLabs/fuels-ts/blob/06aadf6e1efa1bd8701c0e55e18f99d81aee9263/packages/fuel-gauge/src/predicate/predicate-configurables.test.ts#L211-L214
> 
> Instead, you should use `FuelError`, as you did on the other occurrences:
> 
> https://github.com/FuelLabs/fuels-ts/blob/06aadf6e1efa1bd8701c0e55e18f99d81aee9263/packages/fuel-gauge/src/predicate/predicate-configurables.test.ts#L230-L233

This way, the tests isn’t pass.

Reply:
I committed the change you mentioned. Take a look at CI

Reply:
@msensys the test is failing because of [this line](https://github.com/FuelLabs/fuels-ts/blob/e2259ed5186c9909f324f20342b8ce259cee685f/packages/account/src/predicate/predicate.ts#L254) catching and rethrowing the error with an updated message. To fix the failing test, change the expected message in your test to `Error setting configurable constants: Predicate has no configurable constants to be set.`; you were missing the `Error setting configurable constants: ` part. We should maybe rethink this try/catch here, but it's out of the scope of this issue. Just changing the expected message is enough.